### PR TITLE
docs(ingestion): add data programatically - EUBFR-127

### PR DESCRIPTION
**Pull request by kalinchernev**
Demos are showing examples with a rest client, but the notes are hopefully clear enough we speak HTTP protocol which can be used by any other tool.

### CODE REVIEW

File: docs/HOW_TO_TEST.md [hunk 1]
```diff
@@ -53,12 +56,14 @@ For more information about access key setup, see [this serverless guide](https:/
...
+
+* `yarn`
 
 Deploy on root
-  - go to root of the project
-  - `yarn deploy`
-  - `yarn deploy-demo`
+
+* go to root of the project
+* `yarn deploy`
```
Comment (yhuard) [inline docs/HOW_TO_TEST.md]: Nothing about `yarn deploy-services`?

File: docs/PUSHING_DATA.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+For example, given a root endpoint `API` being `https://ti5rsoocwg.execute-api.eu-central-1.amazonaws.com/test/`, the following endpoints related to storage management of ingestion data:
+
+| #   | Operation                    | Endpoint                   | Method | Headers (on top of AWS signature) |
+| --- | ---------------------------- | -------------------------- | ------ | --------------------------------- |
+| 1   | Upload                       | {`API`}/storage/signed_url | GET    | `x-amz-meta-producer-key`         |
+| 2   | Download                     | {`API`}/storage/download   | GET    | `x-amz-meta-computed-key`         |
+| 3   | Update                       | {`API`}/storage/update     | PUT    | -                                 |
+| 4   | Delete                       | {`API`}/storage/delete     | GET    | `x-amz-meta-computed-key`         |
+| 5   | Get a list of files          | {`API`}/meta-index/list    | GET    | -                                 |
```
Comment (yhuard) [inline docs/PUSHING_DATA.md]: `/meta-index/list` and `/meta-index/file` don't exist anymore

Comment (yhuard) [reply docs/PUSHING_DATA.md]: Use ES instead

**Review by yhuard — CHANGES_REQUESTED** [review]
The part about the meta-index API has to be updated since ES has replaced it

File: docs/HOW_TO_TEST.md (hunk 1 above)
Comment (kalinchernev) [reply docs/HOW_TO_TEST.md]: Good point! For this, I just made a reference to the main README, which is the best source to explain this part.

**Review by yhuard — APPROVED** [review]
Looks better now

**Pull request merged**